src/main/resources/application.yaml

```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/happening-now
```

Comment:
Use environment variables in order to receive these values.
It goes for: Hostname, Port, Username and Password